jme3-core/src/main/java/com/jme3/shadow/AbstractShadowRenderer.java

```diff
@@ -200,7 +204,7 @@ protected void initForcedRenderState() {
      *
      * @param postShadowMat
      */
-    protected final void setPostShadowMaterial(Material postShadowMat) {

```

Comment:
I'm not sure why I set this method to final in the first place, I don't think it will make a huge difference.
However, I'm really wondering why you need this method to no be final..?